openmdao/recorders/sqlite_recorder.py

```diff
@@ -32,7 +32,7 @@ def record_metadata(self, group):
         Args
         ----
         group : `System`
-            `System` containing vectors 
+            `System` containing vectors
```

Comment:
This file was apparently inadvertently opened to review things, and saved somewhere along the way with Atom fixing this extra space.